README.md

```diff
@@ -0,0 +1,89 @@
+| openvpn_bridge                     |                                                  | Bridge settings to be configured for defaults see below                                                                                                         |
+| openvpn_dns_servers                | `["8.8.8.8","8.8.4.4"]`                          | List of DNS servers to push to the client                                                                                                                       |
+| openvpn_cipher                     | `AES-256-CBC`                                    | Cipher to use.                                                                                                                                                  |
+| openvpn_auth_hash_algo             | `SHA256`                                         | Algorithm to use for auth.                                                                                                                                      |
+| openvpn_openssl_digest             | `sha256`                                         | Digest Algorithm to use when signing and creating certs.                                                                                                        |
+| openvpn_openssl_days               | `3650`                                           | How many days are the certs valid.                                                                                                                              |
+| openvpn_use_lzo                    | `true`                                           | Enable or disable compression.                                                                                                                                  |
+
+### LDAP object
+
+| Variable            | Default / Mandatory                       | Description                                                                                    |
+|---------------------|-------------------------------------------|------------------------------------------------------------------------------------------------|
+| url                 | `ldap://host.example.com`                 | Address of you LDAP backend with syntax ldap[s]://host[:port]                                  |
+| anonymous_bind      | `False`                                   | This is not an Ansible boolean but a string that will be pushed into the configuration file |
+| bind_dn             | `uid=Manager,ou=People,dc=example,dc=com` | Bind DN used if "anonymous_bind" set to "False"                                                |
+| bind_password       | `mysecretpassword`                        | Password of the bind_dn user                                                                   |
+| tls_enable          | `no`                                      | Force TLS encryption. Not necessary with ldaps addresses                                       |
```

Comment:
So could you say that this enables STARTTLS instead of saying it forces TLS encryption?

Reply:
fixed